index.html

```diff
@@ -154,5 +154,31 @@ <h3>About me</h3>
             </div>
         </div>
     </section>
+    <section class="contact">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+            <input type="text" id="fullName" name="fullName" placeholder="Full Name" required>
+            <input type="text" id="firstName" name="firstName" placeholder="First Name" required>
+            <input type="text" id="lastName" id="lastName" placeholder="Last Name" required>
+            <input type="email" id="email" name="email" placeholder="Email Address" required>
+            <textarea id="message" id="message" placeholder="Hey, I'd like to connect to..."></textarea>
```

Comment:
In the textarea field, you need to add two things according to the requirements.

1. Just like above comment, you need to add `maxlength="500"` attr in the textarea field. 
2. And you need to add `required` attr to disable the submit btn when the field is empty for this requirement: [A user cannot submit the form if one or more of the fields is empty.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#general-requirements)